lib/cleanup/helpers/language.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+# Docker Helpers for Plugin Class
+# Its main functions are to:
+#  1. Set Items to be binded/mounted to Plugin Container
+#     - Include script if script type
+#     - Include plugin config if configured with path to mount/bind
+#  2. Set Script Language based on plugin config
+#     - Language determines which image is pulled for script exec
+class Cleanup
```

Comment:
Curious if this should be moved to a reusable location for both Plugins and Cleanup to not repeat?